wifiphisher/pywifiphisher.py

```diff
@@ -624,9 +630,13 @@ def cb(pkt):
                     if args.accesspoint not in [pkt.addr1, pkt.addr2]:
                         return
 
+                # don't deauth our own AP
+                if ap_mac in [pkt.addr1, pkt.addr2]:
```

Comment:
Have you tested this? E.g. are you able to access/print `ap_mac` there? I believe you will have to do a `global ap_mac` to access it there.